.circleci/config.yml

```diff
@@ -68,6 +68,31 @@ jobs:
           working_directory: ios/PurchasesHybridCommon
           path: test_output
           destination: scan-output
+
+  integration-test-ios:
+    executor: ios-executor
+    steps:
+      - checkout
```

Comment:
[not for this PR] we'll also likely end up having a lot of shared circleci code, maybe we could reuse the shared repo for the fastlane plugin for that too? 
what do you think? //cc @tonidero since you're working on sharing code and @vegaro since you've done some work with shared orbs in the past

Reply:
Great point 👍🏻

Reply:
Ah I missed this. I agree with moving all the fastlane logic we can to the new plugin. About moving circleci logic, I'm not sure if using that same repo would be the cleanest approach. In a 5-min google search, I didn't find an easy way to share this configuration across repos, but we can study what options there are

Reply:
maybe we can have a [separate repo for orbs](https://circleci.com/docs/orb-author#orb-development-kit)

Reply:
Ahh right, that's nice! We can probably use that yeah 👍